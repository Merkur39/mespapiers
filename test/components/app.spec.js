'use strict'

/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'

import AppLike from '../components/AppLike'
import App from '../../src/components/App'

describe('App component only', () => {
  it('should be mounted correctly', () => {
    const component = shallow(
      <AppLike>
        <App />
      </AppLike>
    ).getElement()
    expect(component).toMatchSnapshot()
  })
})
