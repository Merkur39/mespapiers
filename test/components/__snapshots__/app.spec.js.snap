// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component only should be mounted correctly 1`] = `
<CozyProvider
  client={
    CozyClient {
      "uri": undefined,
    }
  }
>
  <I18n
    defaultLang="en"
    dictRequire={[Function]}
    lang="en"
  >
    <BreakpointsProvider>
      <HashRouter>
        <Unknown />
      </HashRouter>
    </BreakpointsProvider>
  </I18n>
</CozyProvider>
`;
